Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/SelectStatisticsData.tsx

```diff
@@ -40,38 +40,13 @@ declare let userIsAdmin: boolean;
 //Todo
 // # Removed Meter table for now due to EPRI Time contraints
 
-type Duration = ('Custom' | '1 Day' | '7 Days' | '30 Days' | '90 Days' | '180 Days' | '365 Days')
-
 const SelectStatisticsData = () => {
     const dispatch = useDispatch();
 
     const dateRange = useSelector(SelectStatisticsrange)
     const dataFilter = useSelector(SelectStatisticsFilter)
 
-    const [range, setRange] = React.useState<Duration>('Custom');
-    
-    React.useEffect(() => {
-        if (range === 'Custom')
-            return;
-        const toTime: moment.Moment = moment(dateRange.start, "YYYY-MM-DD").add(GetDays(range), 'days');
-        dispatch(updateStatisticsRange({ ...dateRange, end: toTime.format("YYYY-MM-DD") }))
-    }, [range]);
-
-    function GetDays(val: Duration) {
-        if (val === '1 Day')
-          return 1;
-        if (val === '7 Days')
-          return 7;
-        if (val === '30 Days')
-          return 30;
-        if (val === '90 Days')
-          return 90;
-        if (val === '180 Days')
-          return 180;
-        if (val === '365 Days')
-          return 365;
-        return 0;
-      }
+    type TimeUnit = 'y' | 'M' | 'w' | 'd' | 'h' | 'm' | 's' | 'ms'
```

Comment:
We should move this outside of the React component so it doesn't recreate every render.